source/includes/_payment.md

```diff
@@ -2090,9 +1326,7 @@ class Payment {
    "auditLogs":[]
 }
 ```
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?